quill-sql/src/main/scala/io/getquill/sources/sql/idiom/SqlIdiom.scala

```diff
@@ -204,8 +204,10 @@ trait SqlIdiom {
 
   implicit def actionShow(implicit strategy: NamingStrategy): Show[Action] = {
 
-    def set(assignments: List[Assignment]) =
-      assignments.map(a => s"${strategy.column(a.property)} = ${scopedShow(a.value)}").mkString(", ")
+    def set(assignments: List[Assignment], properties: List[PropertyAlias]) = {
```

Comment:
I have reviewed `CqlIdiom` and it suffers from the same problem. For all SQL-like idioms, bouldn't be better to extract this into a `RenameAssignments` class/module that we can reuse it? E.g. a class that accepts the NamingStrategy in its constructor and that provides the method `set`.